run-qemu/run.sh

```diff
@@ -114,6 +114,15 @@ then
         "${GITHUB_ACTION_PATH}/print_test_summary.py" -s "${GITHUB_STEP_SUMMARY}" -j "${KERNEL_TEST}.json"
     fi
 fi
+
+# Try to collect $IMG:/command_output to OUTPUT_DIR
+if [[ -n ${OUTPUT_DIR} ]]
```

Comment:
Should `${OUTPUT_DIR}` be in double quotes? Same question below.

Reply:
Apparently it works both with and without quotes, I changed the code to quoted version.